tpm2tools/keys.go

```diff
@@ -273,7 +283,27 @@ func (k *Key) Unseal(in *tpmpb.SealedBytes, cOpt CertifyOpt) ([]byte, error) {
 		creationHash := sessionHashAlg.New()
 		creationHash.Write(in.GetCreationData())
 		if _, _, err = tpm2.CertifyCreation(k.rw, "", sealed, tpm2.HandleNull, nil, creationHash.Sum(nil), tpm2.SigScheme{}, ticket); err != nil {
-			return nil, fmt.Errorf("failed to certify creation: %v", err)
+			// There is a bug in some older vTPMs, where they are unable to
+			// CertifyCreation when using a Null signing handle (despite this
+			// being allowed by all versions of the TPM spec). To work around
+			// this bug, we use a temporary signing key and ignore the signed
+			// result. To reduce the cost of this workaround, we use a cached
+			// ECC signing key.
+			// The bug will cause the TPM to return a 0x1A error code. We will
+			// only try the workaround when we got the error code.
+			paramErr, ok := err.(tpm2.ParameterError)
+			if ok && paramErr.Code == 0x1A {
```

Comment:
Use `tpm2.RCInsufficient` instead of the 0x1A literal.

Reply:
Done